datasets/multilingual_librispeech/multilingual_librispeech.py

```diff
@@ -0,0 +1,153 @@
+import datasets
+from datasets.tasks import AutomaticSpeechRecognition
+
+
+_CITATION = """\
+@article{Pratap2020MLSAL,
+  title={MLS: A Large-Scale Multilingual Dataset for Speech Research},
+  author={Vineel Pratap and Qiantong Xu and Anuroop Sriram and Gabriel Synnaeve and Ronan Collobert},
+  journal={ArXiv},
+  year={2020},
+  volume={abs/2012.03411}
+}
+"""
+
+_DESCRIPTION = """\
+Multilingual LibriSpeech (MLS) dataset is a large multilingual corpus suitable for speech research. The dataset is derived from read audiobooks from LibriVox and consists of 8 languages - English, German, Dutch, Spanish, French, Italian, Portuguese, Polish.
+"""
+
+_URL = "http://www.openslr.org/94"
+_DL_URL_FORMAT = "https://dl.fbaipublicfiles.com/mls/mls_{}.tar.gz"
+
+
+class MultilingualLibrispeechConfig(datasets.BuilderConfig):
+    """BuilderConfig for MultilingualLibrispeech."""
+
+    def __init__(self, name, **kwargs):
+        """
+        Args:
+          name: `string`, name of dataset config
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(MultilingualLibrispeechConfig, self).__init__(
+            version=datasets.Version("2.1.0", ""), name=name, data_dir=_DL_URL_FORMAT.format(name), **kwargs
+        )
+
+
+class MultilingualLibrispeech(datasets.GeneratorBasedBuilder):
+    """Multilingual Librispeech dataset."""
+
+    BUILDER_CONFIGS = [
+        MultilingualLibrispeechConfig(name="german", description="German LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="dutch", description="Dutch LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="french", description="French LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="spanish", description="Spanish LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="italian", description="Italian LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="portuguese", description="Portuguese LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="polish", description="Polish dataset"),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "audio": datasets.features.Audio(sampling_rate=16_000),
+                    "text": datasets.Value("string"),
+                    "speaker_id": datasets.Value("int64"),
+                    "chapter_id": datasets.Value("int64"),
+                    "id": datasets.Value("string"),
+                }
+            ),
+            supervised_keys=("file", "text"),
+            homepage=_URL,
+            citation=_CITATION,
+            task_templates=[AutomaticSpeechRecognition(audio_file_path_column="file", transcription_column="text")],
+        )
+
+    def _split_generators(self, dl_manager):
+        archive_path = dl_manager.download_and_extract(self.config.data_dir)
```

Comment:
Just we should not forget that this should be refactored to use `iter_archive` once merged:
- #3110

Reply:
ok, but this is not relevant for this PR no?